docs/system_setup.rst

```diff
@@ -0,0 +1,390 @@
+System setup
+============
+
+To use ewatercycle package you need to setup the system with software
+and data.
+
+This chapter is for system administators or Research Software Engineers who need to setup a system for the eWatercycle platform.
+
+The setup steps:
+
+1.  `Conda environment <#conda-environment>`__
+2.  `Install ewatercycle package <#install-ewatercycle-package>`__
+3.  `Configure ESMValTool <#configure-ESMValTool>`__
+4.  `Download climate data <#download-climate-data>`__
+5.  `Install container engine <#install-container-engine>`__
+6.  `Configure ewatercycle <#configure-ewatercycle>`__
+7.  `Model container images <#model-container-images>`__
+8.  `Download example parameter sets <#download-example-parameter-sets>`__
+9.  `Prepare other parameter sets <#prepare-other-parameter-sets>`_
+10. `Download example forcing <#download-example-forcing>`__
+11. `Download observation data <#download-observation-data>`__
+
+Conda environment
+-----------------
+
+The ewatercycle Python package uses a lot of geospatial dependencies
+which can be installed easiest using `Conda <https://conda.io/>`__ package
+management system.
+
+Instal Conda by using the `miniconda
+installer <https://docs.conda.io/en/latest/miniconda.html>`__. For more thourough instuctions see `HPC to cluster <hpc_to_cluster.rst>`_ chapter.
```

Comment:
I think that the instructions in HPC to cluster should be removed from this part. They are not about conda installation.

Reply:
There is a install conda section at https://ewatercycle.readthedocs.io/en/latest/hpc_to_cluster.html#first-run-preparations, which I tried to point to. Now updated hpc_to_cluster.rst with same install eWaterCycle package instructions